submissions/Harnytskyi/task_js_dom/css/style.css

```diff
@@ -0,0 +1,143 @@
+.nav_ul {
+    list-style: none;
+    padding-inline-start: 0;
+}
+
+.nav-open-button {
+    display: none;
+    background: no-repeat;
+    cursor: pointer;
+    border: none;
+    width: 50px;
+    height: 50px;
+}
+
+.nav-open-button:focus {
+    outline: none;
+    border: 2px solid white;
+    border-radius: 5px;
+}
+
+.header {
+    background-color: #013368;
+    display: flex;
+}
+
+.header-image {
+    max-height: 100px;
+}
+
+.header-title {
+    color: white;
+    font-size: 50px;
+    text-align: center;
+    align-self: center;
+}
+
+.nav {
+    display: flex;
+    flex-direction: column;
+    width: 260px;
+}
+
+.nav-button {
```

Comment:
add `cursor: pointer;` for the `hover` state of this button. It would be better for the UX of your app.